dht.go

```diff
@@ -430,62 +430,67 @@ func (dht *IpfsDHT) fixLowPeersRoutine(proc goprocess.Process) {
 			return
 		}
 
-		if dht.routingTable.Size() > minRTRefreshThreshold {
-			continue
-		}
+		dht.fixLowPeers(dht.Context())
+	}
 
-		// we try to add all peers we are connected to to the Routing Table
-		// in case they aren't already there.
-		for _, p := range dht.host.Network().Peers() {
-			dht.peerFound(dht.Context(), p, false)
-		}
+}
 
-		// TODO Active Bootstrapping
-		// We should first use non-bootstrap peers we knew of from previous
-		// snapshots of the Routing Table before we connect to the bootstrappers.
-		// See https://github.com/libp2p/go-libp2p-kad-dht/issues/387.
-		if dht.routingTable.Size() == 0 {
-			if len(dht.bootstrapPeers) == 0 {
-				// No point in continuing, we have no peers!
-				continue
-			}
+// FixLowPeers attempts to connect to bootstrap peers in order to populate the routing table
```

Comment:
This comment isn't quite accurate as the function does more than just connecting to bootstrap peers and generally doesn't even do that.

Perhaps reuse the comment from above and make it "fixLowPeers tries to get more peers into the routing table if we're below the threshold". The `fixLowPeersRoutine` function is then small enough that we could probably drop the comment or make it something like "fixLowPeersRouting manages simultaneous requests to fixLowPeers"